mantidproject/mantid
# Modify output xyz file names produced by Focus in Pearl diffraction scripts

**Description of work.**

When a Focus is run using the Pearl powder diffraction scripts there are some files output in an XYZ file format which have filenames like:

PRL113355-113361_tt70_tof_xye-0.dat
PRL113355-113361_tt70_d_xye-0.dat

These have been modified as requested to be of the shorter form:

PRL113355-113361_tt70_tof_0.xyz
PRL113355-113361_tt70_d-0.xyz

**Report to:** Nick Funnell

**To test:**

Unzip the attached zip file into a folder.
Adjust paths in the test.py file to point at the folder you've just created
Also adjust path in the user_experiment.yaml file
Run test.py from Workbench (will need archive access enabled)
Observe that files with the new filename format are produced in the 16_4\Danny\ subfolder

[PearlFocusTest.zip](https://github.com/mantidproject/mantid/files/5326181/PearlFocusTest.zip)


Fixes #29421 
.



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
